src/breadcrumbs/index.cjsx

```diff
@@ -47,39 +48,48 @@ Breadcrumbs = React.createClass
     task: tasks.get(taskId)
     moduleInfo: tasks.getModuleInfo(taskId)
 
-  render: ->
-    {task, moduleInfo} = @state
-    {currentStep, canReview} = @props
-
-    return null if _.isEmpty(task.steps)
-
-    crumbs = _.map(task.steps, (crumbStep, index) ->
-      crumb =
-        key: index
-        data: crumbStep
-        crumb: true
-        type: 'step'
-    )
+  makeCrumbEnd: (type, enabled) ->
+    {moduleInfo} = @state
 
     reviewEnd =
       type: 'end'
-      key: crumbs.length
       data:
-        id: ''
+        id: "#{type}"
         title: moduleInfo.title
-      disabled: not canReview
+      className: "coach-#{type}"
```

Comment:
Hmm, where else do we dynamically create class names? (maybe this should be a `data-type` attribute so it allows an enum?)